jax_distribution/systems/ff_ippo_rware.py

```diff
@@ -229,8 +229,8 @@ def _loss_fn(
                 loss_info, grads = grad_fn(params, opt_state, traj_batch, advantages, targets)
 
                 # Compute the parallel mean (pmean) over the batch.
-                # This calculation is inspired by the Anakin architecture demo.
-                # This pmean could be a regular mean as the batch axis is on all devices.
+                # This calculation is inspired by the Anakin architecture demo notebook.
```

Comment:
Similarly a comment underneath here with a link to the notebook: 
`# available at https://tinyurl.com/26tdzs5x`